lib/sycamore/sycamore/transforms/term_frequency.py

```diff
@@ -0,0 +1,39 @@
+from collections import Counter
+from sycamore.data.document import Document
+from sycamore.functions.tokenizer import Tokenizer
+from sycamore.plan_nodes import Node, NonGPUUser, SingleThreadUser, Transform
+from sycamore.transforms.map import Map
+
+
+def compute_term_frequency(document: Document, tokenizer: Tokenizer, with_token_ids: bool) -> Document:
+    tokens = tokenizer.tokenize(document.text_representation or "", as_ints=with_token_ids)
```

Comment:
May want to explain somewhere that this is for post-exploded Documents and not Elements.

Reply:
yeah we need an easy way to tell pre/post explode transforms apart

Reply:
there's a whole bunch of transform-ordering soft-validations we can do. Like, 
explode -> sketch -> augment_text -> embed -> term_frequency 
makes a lot more sense than
explode -> embed -> term_frequency -> augment_text -> sketch